src/scenes/ProgressReports/EditForm/ProgressReportStepper.tsx

```diff
@@ -75,15 +75,19 @@ export const ProgressReportStepper = () => {
         {Object.entries(groupedStepMap).map(([title, steps], index) => (
           <div key={title}>
             {index > 0 && <StepConnector sx={singleConnectorSx} />}
-            <Typography sx={typographySx}>{title}</Typography>
+            <Typography sx={typographySx}>
+              {title === 'Narrative Report' ? 'Investor Connection' : title}
+            </Typography>
             <Stepper orientation="vertical" sx={stepperSx}>
               {steps.map(([label, StepComp]) => (
                 <Step
                   key={label}
                   onClick={() => setProgressReportStep(label)}
                   active={StepComp === CurrentStep}
                 >
-                  <StepButton icon={' '}>{label}</StepButton>
+                  <StepButton icon={' '}>
+                    {label === 'Community Story' ? 'Story' : label}
+                  </StepButton>
```

Comment:
Change here instead
https://github.com/SeedCompany/cord-field/blob/f5e48da9cf6307a00fc2ce8b7691ed1b65b30223/src/scenes/ProgressReports/EditForm/Steps/index.ts#L14-L16

Reply:
Well, I did most of those the hard way.  Making the changes now